src/Http/Http.Results/src/Results.cs

```diff
@@ -502,26 +525,36 @@ public static IResult UnprocessableEntity(object? error = null)
         /// <param name="detail">The value for <see cref="ProblemDetails.Detail" />.</param>
         /// <param name="instance">The value for <see cref="ProblemDetails.Instance" />.</param>
         /// <param name="statusCode">The status code.</param>
-        /// <param name="title">The value for <see cref="ProblemDetails.Title" />.</param>
+        /// <param name="title">The value for <see cref="ProblemDetails.Title" />. Defaults to "One or more validation errors occurred."</param>
         /// <param name="type">The value for <see cref="ProblemDetails.Type" />.</param>
+        /// <param name="extensions">The value for <see cref="ProblemDetails.Extensions" />.</param>
         /// <returns>The created <see cref="IResult"/> for the response.</returns>
         public static IResult ValidationProblem(
             IDictionary<string, string[]> errors,
             string? detail = null,
             string? instance = null,
             int? statusCode = null,
             string? title = null,
-            string? type = null)
+            string? type = null,
+            IDictionary<string, object?>? extensions = null)
         {
             var problemDetails = new HttpValidationProblemDetails(errors)
             {
                 Detail = detail,
                 Instance = instance,
-                Title = title,
+                Title = title ?? "One or more validation errors occurred.",
                 Type = type,
                 Status = statusCode,
             };
```

Comment:
Nit: so we don't have to remember to change this string in two places if we ever decide to.

```suggestion
                Type = type,
                Status = statusCode,
            };
            
            problemDetails.Title = title ?? problemDetails.Title;
```